wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -274,3 +274,68 @@ def test_items_dict_4(self):
             wordpress_item.get_yoast_description_value(),
             "This page has a default description",
         )
+
+
+class WordpressImporterTestsParsePostMeta(TestCase):
```

Comment:
**Question**: What value of `settings.WORDPRESS_IMPORT_ITEMS_TO_CACHE` is used in these tests? Can we include a test that if there are two entries there, both are used? This is related to my comments on the ItemCache `.process()` and `._get_hook_handler_data()` methods.

Reply:
Here there is no `settings.WORDPRESS_IMPORT_ITEMS_TO_CACHE` in play. It's testing the new method for populating `wp_post_meta` JSONField content. wp_post_meta is always saved do you think we should only save that if there is `settings.WORDPRESS_IMPORT_ITEMS_TO_CACHE`?

Reply:
@nimasmi I think this is the Issue and the previous one that we talked about earlier today.

The class  ItemsCache approach is quite different in the Import Authors PR as I used your ideas for new the class TagsCache() for authors.

**Question** Should I pick out those changes and implement them here? There are quite a lot of changes that would complicate this PR though.